oecophylla/taxonomy/taxonomy.rule

```diff
@@ -149,25 +147,25 @@ rule taxonomy_bracken:
             report_base = os.path.basename(output['report'])
 
             shell("""
-                  set +u; {params.kraken_env}; set -u
+                  set +u; {params.env}; set -u
 
                   kraken {input.forward} {input.reverse} \
-                    --db {params.kraken_db} \
+                    --db {params.db} \
                     --paired \
                     --fastq-input \
                     --gzip-compressed \
                     --only-classified-output \
-                    --threads {threads} \
+                    --threads {params.threads} \
                     1> {temp_dir}/map.tmp \
                     2> {log}
 
                   kraken-report {temp_dir}/map.tmp \
-                    --db {params.kraken_db} \
+                    --db {params.db} \
                     1> {temp_dir}/report.tmp \
                     2>> {log}
 
                   est_abundance.py -i {temp_dir}/report.tmp \
-                    -k {params.bracken_db} \
+                    -k {params.kmers} \
                     -o {temp_dir}/{table_base} \
                     2>> {log} 1>&2
 
```

Comment:
Is there a combine bracken tables rule?

Reply:
No. I guess @sjanssen2 's parser script will do that.

Reply:
ah right, see that now. thanks!